kolibri/core/api.py

```diff
@@ -33,3 +39,17 @@ def finalize_response(self, request, response, *args, **kwargs):
         response = super(CSVModelViewSet, self).finalize_response(request, response, *args, **kwargs)
         response['content-disposition'] = 'attachment; filename=%s.csv' % self.csv_export_filename
         return response
+
+
+class InfoViewSet(viewsets.ViewSet):
+
+    def list(self, request):
```

Comment:
Probably best to just have this be a detail endpoint? Since it's just one object being returned?